### ISSUE TITLE
TypeScript: Cannot find module 'react-helmet'

### ISSUE THREAD

**Issue by dangodev** [staylor/react-helmet-async#38]
```
✖ ｢atl｣: Checking finished with 1 errors
[at-loader] ./node_modules/react-helmet-async/index.d.ts:2:38
    TS2307: Cannot find module 'react-helmet'.
```

The TS definition seems to require `react-helmet` even though it’s not a dependency or peer dep. Any way we can use this package without having to install `react-helmet`?

**Comment by obedm503** [comment]
you don't really have to install `react-helmet`. It works  perfectly well if you just install `@types/react-helmet`

**Comment by staylor** [comment]
this should be fixed by now after #83
